aws/aws-env.sh

```diff
@@ -3,7 +3,7 @@ show_help() { cat <<'EOF'
 aws-env: Wrapper for AWS temporary sessions using MFA and roles
 ===============================================================
 
-Copyright (c) 2017 FP Complete Corp.  
+Copyright (c) 2017 FP Complete Corp.
 Author and maintainer: Emanuel Borsboom <manny@fpcomplete.com>
```

Comment:
The whitespace at the end of the line here is significant and shouldn't be removed (it causes markdown to insert a newline).